bench/workgen/runner/many-dhandle-stress.py

```diff
@@ -90,6 +90,7 @@
 workload.options.max_latency=1000
 workload.options.sample_rate=1
 workload.options.sample_interval_ms = 5000
+workload.options.max_idle_table_cycle = 2
```

Comment:
I think it will be useful to take the latest updates from many-dhandle-stress.wtperf, including the comments. So for instance it will be useful in the future to have the commented out "max_idle_table_cycle_fatal=true":

```
# Uncomment to fail instead of generating a warning
# workload.options.max_idle_table_cycle_fatal=true
```